etc/behat/suites/ui/product/managing_product_attributes.yml

```diff
@@ -0,0 +1,20 @@
+# This file is part of the Sylius package.
+# (c) Paweł Jędrzejewski
+
+default:
+    suites:
+        ui_managing_product_attributes:
+            contexts_as_services:
+                - sylius.behat.context.hook.doctrine_orm
+
+                - sylius.behat.context.setup.security
+                - sylius.behat.context.setup.locale
+                - sylius.behat.context.setup.product_attribute
+
+                - sylius.behat.context.transform.locale
```

Comment:
Please put transformers before setup. And also, can you sort them alphabetically?